package.json

```diff
@@ -84,10 +89,12 @@
           "type": "string",
           "enum": [
             "babylon",
-            "flow"
+            "flow",
+            "typescript",
+            "postcss"
```

Comment:
I think we should only suggest babylon/flow for JS.
The other one are inferred from language type ?
Or am I missing something?